doc/man7/Ed25519.pod

```diff
@@ -49,7 +49,7 @@ Ed25519 or Ed448 public keys can be set directly using
 L<EVP_PKEY_new_raw_public_key(3)> or loaded from a SubjectPublicKeyInfo
 structure in a PEM file using L<PEM_read_bio_PUBKEY(3)> (or similar function).
 
-Ed25519 and Ed448 can be tested within L<speed(1)> application since version 1.1.1.
+Ed25519 and Ed448 can be tested within L<openssl-speed(1)> application since version 1.1.1.
```

Comment:
within **the** L<openssl-speed(1)> application

Reply:
"with the"  fixed.